code/modules/surgery/facial_surgery.dm

```diff
@@ -0,0 +1,275 @@
+		"You are beginning to cauterize the incision on [target]'s face and neck with \the [tool].")
+		..()
+
+	end_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+		var/obj/item/organ/external/affected = target.get_organ(target_zone)
+		user.visible_message("<span class='notice'>[user] cauterizes the incision on [target]'s face and neck with \the [tool].</span>", \
+		"<span class='notice'>You cauterize the incision on [target]'s face and neck with \the [tool].</span>")
+		affected.open = 0
+		affected.status &= ~ORGAN_BLEEDING
+		if (target.op_stage.face == 3)
+			var/obj/item/organ/external/head/h = affected
+			h.disfigured = 0
+		target.op_stage.face = 0
+
+	fail_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+		var/obj/item/organ/external/affected = target.get_organ(target_zone)
+		user.visible_message("<span class='warning'>[user]'s hand slips, leaving a small burn on [target]'s face with \the [tool]!</span>", \
+		"<span class='warning'>Your hand slips, leaving a small burn on [target]'s face with \the [tool]!</span>")
+		target.apply_damage(4, BURN, affected)
+
+
+/datum/surgery_step/robotics/face
+	priority = 2
+	can_infect = 0
+	can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+		return target_zone == "mouth"
+
+/datum/surgery_step/robotics/face/synthskinopen
+	allowed_tools = list(
+	/obj/item/weapon/scalpel = 100,
+	/obj/item/weapon/material/knife = 75,
+	/obj/item/weapon/material/shard = 50
+	)
+
+	min_duration = 90
+	max_duration = 110
+
+	can_use(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+		return ..() && target.op_stage.face == 0 && target.get_species() == "Shell Frame"
+
+	begin_step(mob/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+		user.visible_message("[user] starts to cut open [target]'s synthskin face and neck with \the [tool].", \
+		"You start to cut open [target]'s synthskin face and neck with \the [tool].")
+		..()
+
+	end_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+		user.visible_message("<span class='notice'>[user] has cut open [target]'s synthskin face and neck with \the [tool].</span>" , \
+		"<span class='notice'>You have cut open [target]'s synthskin face and neck with \the [tool].</span>",)
+		target.op_stage.face = 1
+
+	fail_step(mob/living/user, mob/living/carbon/human/target, target_zone, obj/item/tool)
+		var/obj/item/organ/external/affected = target.get_organ(target_zone)
+		user.visible_message("<span class='warning'>[user]'s hand slips, slicing [target]'s throat wth \the [tool]!</span>" , \
+		"<span class='warning'>Your hand slips, slicing [target]'s throat wth \the [tool]!</span>" )
+		affected.createwound(CUT, 60)
+
+
+/datum/surgery_step/robotics/face/prepare_face
+	allowed_tools = list(
+	/obj/item/weapon/retractor = 100,
+	/obj/item/weapon/material/hatchet/tacknife = 75
```

Comment:
This does not work, and results in amputation, IIRC.

Reply:
https://puu.sh/CkWuI/912b7cfab6.mp4

You might of been targeting the patients head, the plastic surgery method doesnt amputate if you target the mouth